package.json

```diff
@@ -47,7 +47,8 @@
   },
   "dependencies": {
     "@phenomnomnominal/tsquery": "^3.0.0",
-    "babel-runtime": "^6.22.0"
+    "babel-runtime": "^6.22.0",
+    "typescript": "~3.2.4"
```

Comment:
Looks like a mistake

Reply:
Hey, no it isn't, `tsquery` requires it, we missed this requirement because `WSR` had installed `typescript` for dev purposes as well, so it worked.
But running `import-path` locally on a dummy folder wouldn't work.
Sorry it was mixed in the same commit.
I can also move it to being a peerDep if you prefer that.

Reply:
so please add it as a peerDep